docs/docs/NewAPIGuide/Layers/core.md

```diff
@@ -0,0 +1,71 @@
+## Dense ##
```

Comment:
Make it clear that we now provide Keras 1.2 style APIs

Reply:
Specified in https://github.com/intel-analytics/BigDL/pull/2342/files#diff-e55d3c2fc149d4f9b82b3db36e760e5eR3